app/models/post.rb

```diff
@@ -1,8 +1,28 @@
 class Post < ApplicationRecord
   include Elasticsearch::Model
+  include PgSearch::Model
 
   validates :title, :body, :topic, presence: true
 
+  # Full-text search
+  # multisearchable against: [:title, :body, :topic]
+  pg_search_scope(
+    :search_full_text,
+    against: {
+      body: 'A',
```

Comment:
just curious.. what are these 'A' 'B' 'C' ?

Reply:
This is for [weighting](https://github.com/Casecommons/pg_search#weighting)